Plots/Contours/NCL_conwomap_2.py

```diff
@@ -0,0 +1,73 @@
+"""
+NCL_conwomap_2.py
+===============
+This script illustrates the following concepts:
+   - Drawing a simple filled contour plot
+   - Selecting a different color map
+   - Changing the size/shape of a contour plot using viewport resources
+See https://www.ncl.ucar.edu/Applications/Scripts/conwomap_2.ncl for further information.
+"""
+
+###############################################################################
+# Import packages
+import numpy as np
+import xarray as xr
+import cartopy.crs as ccrs
+from util.make_byr_cmap import make_byr_cmap
+
+import matplotlib.pyplot as plt
+import matplotlib.ticker as tic
+
+###############################################################################
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset("../../data/netcdf_files/cone.nc")
+u = ds.u.isel(time=4)
+
+###############################################################################
+# Plot
+# First get axes for a projection of preference
+fig = plt.gcf()
+projection = ccrs.PlateCarree()
+ax = plt.axes(projection=projection)
+
+# Import an NCL colormap
+newcmp = make_byr_cmap()
+
+# Plot borderlines first
+u.plot.contour(ax=ax, vmin=-1, vmax=10, levels=12, linewidths=0.5, colors='k', add_colorbar=False, transform=projection, extend='neither', add_labels=False)
+# Plot filled contours
+p = u.plot.contourf(ax=ax, vmin=-1, vmax=10, levels=12, cmap=newcmp, add_colorbar=False, transform=projection, extend='neither', add_labels=False)
+
+###############################################################################
+# Adjust figure size and plot parameters to get identical to original NCL plot
+fig = plt.gcf()
+fig.set_size_inches((10, 6))
+
+# Hard-code tic values. This assumes data are global
+ax.set_xticks(np.linspace(0, 40, 5))
+ax.set_yticks(np.linspace(0, 25, 6))
+
+# Adjust axes limits
+ax.set_xlim((0,49))
+ax.set_ylim((0,29))
```

Comment:
Original plot also needs  `ax.set_xlabel("X")` and `ax.set_ylabel("Y")`

Reply:
Done